src/Core/Components/NavMenu/FluentNavMenu.razor.cs

```diff
@@ -222,6 +222,8 @@ private void HandleNavigationManagerLocationChanged(object? sender, LocationChan
         if (menuItem is not null)
         {
             _selectedTreeItem = menuItem.TreeItem;
+            // menuItem.SelectedChanged.InvokeAsync(true);
```

Comment:
Why to add these comments?

Reply:
This was a test to unblock Peter and his work. As he created a new PR, this one will be not merged and closed